backend/app/glossary/models.py

```diff
@@ -11,31 +11,43 @@
     from app.schema import User
 
 
+class ProcessingStatuses:
+    IN_PROCESS = "IN_PROCESS"
+    DONE = "DONE"
+
+
 class GlossaryDocument(Base):
     __tablename__ = "glossary_document"
     id: Mapped[int] = mapped_column(primary_key=True)
-    created_by: Mapped[int] = mapped_column(ForeignKey("user.id"))
-    created_at: Mapped[datetime] = mapped_column(default=datetime.now(UTC))
-    updated_at: Mapped[datetime] = mapped_column(default=datetime.now(UTC))
+    created_at: Mapped[datetime] = mapped_column(default=datetime.now)
+    updated_at: Mapped[datetime] = mapped_column(default=datetime.now)
+
+    original_document: Mapped[str] = mapped_column()
+    processing_status: Mapped[str] = mapped_column(
+        default=ProcessingStatuses.IN_PROCESS
+    )
+    upload_time: Mapped[datetime] = mapped_column(default=datetime.now)
 
     records: Mapped[list["GlossaryRecord"]] = relationship(
         back_populates="document",
         cascade="all, delete-orphan",
         order_by="GlossaryRecord.id",
     )
+    user_id: Mapped[int] = mapped_column(ForeignKey("user.id"))
     user: Mapped["User"] = relationship(back_populates="glossaries")
 
 
 class GlossaryRecord(Base):
     __tablename__ = "glossary_record"
 
     id: Mapped[int] = mapped_column(primary_key=True)
-    created_at: Mapped[datetime] = mapped_column(default=datetime.now(UTC))
-    updated_at: Mapped[datetime] = mapped_column(default=datetime.now(UTC))
+    created_at: Mapped[datetime] = mapped_column(default=datetime.now)
+    updated_at: Mapped[datetime] = mapped_column(default=datetime.now)
 
-    document_id: Mapped[int] = mapped_column(ForeignKey("glossary_document.id"))
-    comment: Mapped[str] = mapped_column()
+    author: Mapped[str] = mapped_column()
```

Comment:
Well, I think it should be mapped to the real user. We just need to understand what to do in case of importing a file. Make it `nullable`? Map it some default user?